sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch

```diff
@@ -89,7 +85,7 @@
     </group>
     <group if="$(arg grasp_controller)">
       <!-- Not available yet -->
-      <!--node name="$(arg hand_id)_grasp_controller" pkg="sr_utilities" type="grasp_controller_spawner.py" output="screen">
+      <!--node name="$(arg hand_prefix)_grasp_controller" pkg="sr_utilities" type="grasp_controller_spawner.py" output="screen">
```

Comment:
should we leave this un-commented code here? in bimanual there isn't ever going to be a hand_prefix, right?